DocumentService/Constants/AzureBlobContainer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DocumentService.Constants
+{
+    public static class AzureBlobContainer
+    {
+
+        public const string DocumentsContainer = "documents";
```

Comment:
Maybe we should move this to the appsettings.json file?

Reply:
Done